src/specs/concepts/permissions_concepts.cpt

```diff
@@ -1,16 +1,70 @@
-# Deploy permissions model contracts from <node>
-* Deploy "PermissionsUpgradable" smart contract setting default account of "Node1" as the guardian account, name this as "upgradable"
-* From <node> deploy "AccountManager" contract passing "upgradable" address, name it "accountmgr"
-* From <node> deploy "OrgManager" contract passing "upgradable" address, name it "orgmgr"
-* From <node> deploy "RoleManager" contract passing "upgradable" address, name it "rolemgr"
-* From <node> deploy "NodeManager" contract passing "upgradable" address, name it "nodemgr"
-* From <node> deploy "VoterManager" contract passing "upgradable" address, name it "votermgr"
-* From <node> deploy "PermissionsInterface" contract passing "upgradable" address, name it "interface"
-* From <node> deploy implementation contract passing addresses of "upgradable", "orgmgr", "rolemgr", "accountmgr", "votermgr", "nodemgr". Name this as "implementation"
-* From <node> execute permissions init on "upgradable" passing "interface" and "implementation" contract addresses
-
-# Create permissions config file and write the file to <nodes>
+# Start permission network <networkType> <version> <id> <qip714block>
+* Start a <networkType> <version> Quorum Network, named it <id>, consisting of "Node1,Node2,Node3"
+* Deploy permissions model contracts from "Node1" with <version>
+* Create permissions config file with <version> and write the file to "Node1,Node2,Node3"
+* Restart network <id>
+* Deploy "storec" smart contract with initial value "5" from a default account in "Node1" until block height reaches <qip714block>
+
+# Validate network admin role
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Validate that org "NWADMIN" is approved, has "Node2" linked and has role "NWADMIN"
+* Validate that org "NWADMIN" is approved, has "Node3" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+
+# Propose new organization to the network <version> <id>
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+* From "Node1" propose and approve new org "ORG1" with "Node4"'s enode id and "Default" account
```

Comment:
Should we make sure that node permissioning is effective? (or maybe this is already being tested somewhere else?)

For example would it make sense here to 
1. start Node4 
2. Validate that Node4 is not permissioned (e.g. by checking if Node4 blocknumber does not increase or checking peers using `admin_peers`)
3. Approve Node4 from Node1
4. Validate that Node4 is permissioned (e.g. by checking that Node4 block number is increasing or checking peers using `admin_peers`)

Reply:
it is covered by the following scenarios:
## Proposing a new organization into the network
## Suspending an organization temporarily
## Deactivating an organization to stop block synchronization in a node

You can refer to the details in permissions_concepts.cpt